src/file-upload.css

```diff
@@ -34,3 +34,82 @@
   50% {transform: translate(-8px)}
   100% {transform: translate(0)}
 }
+
+.files-panel {
```

Comment:
Let's remove `file` or `files` prefixes from names in the `file-upload` namespace.